test/benchmark/main.cc

```diff
@@ -46,6 +47,12 @@ int main(int argc, char** argv) {
     return 0;
   }
 
+  // Reduce logs so benchmark output is readable.
+  Thread::MutexBasicLockable lock;
+  Logger::Context logging_context{spdlog::level::warn, Logger::Context::getFancyLogFormat(), lock,
+                                  false};
+  UNREFERENCED_PARAMETER(logging_context);
```

Comment:
is this UNREFERENCED_PARAMETER necessary? This is the pattern that, for example, LockGuard or MutexLock is used, and we don't need an UNREFERENCED_PARAMETER macro for those.

Reply:
Yeah I wouldn't think this is necessary either.

Reply:
Done